atomicapp/cli/main.py

```diff
@@ -371,7 +371,8 @@ def create_parser(self):
 
     def run(self):
         cmdline = sys.argv[1:]  # Grab args from cmdline
-
+        if len(cmdline) == 0:   # issue 668, add check for no arguments passed
```

Comment:
Thanks for the PR, please remove the issue specific comment and add more generic comment.